### ISSUE TITLE
Startup issue with Tesla graphics card

### ISSUE THREAD

**Issue by ArmcodeZ** [Rem0o/FanControl.Releases#294]
Occurs on version 87 of software, have not tried other versions.
OS version: Windows 10 64-bit

Motherboard ASRock Z270M Extreme4, i7-6700K CPU
Nvidia driver version 27.21.14.6231 (4/8/2021)
Intel GPU is Intel HD Graphics 530 (this works if Tesla is disabled).

My system has two graphics card. The built in Intel GPU and a Tesla M40 24GB (GRID card). 
The Tesla Grid card is NOT capable of video out and cannot support some requests from the Nvidia driver to display.

If I attempt to run the program with both graphics cards I get the following error:
7/17/2021 7:57:14 PM: NvAPIWrapper.Native.Exceptions.NVIDIAApiException: NVAPI_NVIDIA_DEVICE_NOT FOUND
    at NvAPIWrapper.Native.GPUAPI.EnumPhysicalGPUs()
    at NvAPIWrapper.GPU.PhysicalGPU.GetPhysicalGPUs()
   at FanControl.Domain.BackendProviders.NVApiWrapper.NVApiWrapperBackendProvider.Open()
  at System.Collections.Generic.List`1.ForEaach(Action`1 action)
  at FanControl.Domain.ComputerAccessLayer.Initialize()
7/17/2021 7:57:14 PM: Could not initialize.

(Note I typed the above in by hand as I am connected to machine in quesiton via VNC).

I can work around this issue by going into the control panel, disabling the Nvidia card and then starting fan control. It appears in this case that the program will fall back and use the onboard GPU instead of faulting. I can then enable the GPU and run things normally.

Is there a way for this program to support either selecting the graphics method to display (ie. which card is used) via config/command line or support falling back to a second GPU type in the case of the fault I am is experiencing? I do not expect that my Tesla card would support rendering of graphics (under Windows at least). This is a scientific computing card I use for machine learning.

Please let me know if you  need additional information.

**Comment by ArmcodeZ** [comment]
I accidentally submitted the issue early and have updated the initial post. Thank you.

**Comment by Rem0o** [comment]
Did not expect the program to fail at that specific point, since the driver did loaded successfully.  
The line where it is failing is at the point I'm just enumerating the available GPUs before doing anything else, I don't even know yet what GPU you got and it fails, so I can't do much, except unload the NVDIA driver and go on. 
Will cover that case.

**Comment by Rem0o** [comment]
Lookup version 88, the program should be able to load even if the card isn't supported.

**Comment by ArmcodeZ** [comment]
I can confirm that version 88 works around the issue I reported for version 87. On version 88 the program starts as expected. Issue can be closed as fixed.

The log still has an error (if this is relevant to you). I have typed this output in below. 
Output of the log file is:

7/22/2021 7:01:11 PM: NvAPIWrapper.Native.Exceptions.NVIDIAApiException: NVAPI_NVIDIA_DEVICE_NOT_FOUND
   at NvAPIWrapper.Native.GPUApi.EnumPhysicalGPUs()
   at NvAPIWrapper.GPU.PhysicalGPU.GetPhysicalGPUs()
   at FanControl.Domain.BackendProviders.NVApiWrapper.NVApiWrapperBackendProvider:Open()

Note: Even with the above error the program launches as expected (assuming this can be treated as a warning).
